frontend/src/utils/StoryCardUtils.ts

```diff
@@ -0,0 +1,22 @@
+// Change language string to title case
+function changeLanguageToTitleCase(language: string) {
```

Comment:
nit: maybe a name like `convertLanguageTitleCase` works better - `change` can imply modifying the passed in param